pkg/puller/puller.go

```diff
@@ -508,11 +424,14 @@ func peerIntervalKey(peer swarm.Address, bin uint8) string {
 type syncPeer struct {
 	address        swarm.Address
 	binCancelFuncs map[uint8]func() // slice of context cancel funcs for historical sync. index is bin
+	po             uint8
+
+	cursors []uint64
```

Comment:
`cursors` property seems to be unused in `syncPeer` structure, consider removing it.

Reply:
no, we use it :)